src/main/java/bio/terra/service/iam/IamRole.java

```diff
@@ -10,7 +10,9 @@
     CUSTODIAN,
     INGESTER,
     READER,
-    DISCOVERER;
+    DISCOVERER,
+    OWNER,
+    USER;
 
```

Comment:
Do we define these roles anywhere in the codebase? Would it be weird if we did?

Reply:
I'm not sure what you mean. This is in our codebase. We rely on this definition for specifying the roles in the calls to IamService. And we use the actual names of the roles in making requests to Sam.